tracer/test/test-applications/integrations/Samples.MSTestTests/ClassInitializeExceptionTestSuite.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Samples.MSTestTests;
+
+[TestClass]
+public class ClassInitializeExceptionTestSuite
+{
+    public TestContext TestContext { get; set; }
+
+    [ClassInitialize]
+    public static void ClassInitialize(TestContext testContext)
+    {
+        throw new Exception("Class initialize exception");
+    }
+
+    [TestInitialize]
+    public void TestInitialize()
+    {
+        TestContext.WriteLine("Test Initialize");
+    }
+
+    [TestCleanup]
+    public void TestCleanup()
+    {
+        TestContext.WriteLine("Test Cleanup");
+    }
+
+    [ClassCleanup]
+    public static void ClassCleanup()
+    {
+    }
```

Comment:
Do we need tests for exceptions in `[TestInitialize]`, `[TestCleanup]`, `[ClassCleanup]`? And what about in instance/static constructors for the `[TestClass]`? 🤔